asistente_ladm_col/gui/point_spa_uni_cadastre_wizard.py

```diff
@@ -210,7 +210,7 @@ def autodetect_separator(self):
         if os.path.exists(csv_path):
             with open(csv_path) as file:
                 first_line = file.readline()
-                for delimiter in [ item for item in self.known_delimiters ]:
+                for delimiter in self.known_delimiters:
```

Comment:
Ouch, sorry, I do some changes and I forget to remove this.

Reply:
No problem, that's why a pair review is always recommended :D